app/client/cluster_api_client.py

```diff
@@ -76,12 +87,52 @@ def get_build_status(self, build_id):
                                             'URL: {}, Content:{}'.format(build_status_url, response_data))
         return response_data
 
+    def block_until_build_started(self, build_id, timeout=None, build_in_progress_callback=None):
+        """
+        Poll the build status endpoint until the build has started or finished.
```

Comment:
This comment is confusing. Should it be updated or maybe the method should be renamed?